.env.example

```
@@ -0,0 +1,14 @@
+# GitHub access for config-repo
+GIT_USERNAME={깃허브아이디}
+GIT_PASSWORD={깃허브토큰}
+
+# common 라이브러리 다운로드용
+GITHUB_USER={깃허브아이디}
+GITHUB_TOKEN={깃허브토큰}
+
+# Config Server Basic Auth
+CONFIG_SERVER_USERNAME={팀노션 .env 참고}
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)